kb-icon/src/kb-icons.ts

```diff
@@ -1,6 +1,6 @@
 import {LitElement, html, css} from "lit";
 import {customElement, property} from "lit/decorators.js";
-import {icons} from "./icon-list";
+import {icons} from "./icon-list.js";
```

Comment:
Yes, we will fix this in the new webpack conf:)

Reply:
Sounds great :-)